pkg/watch/until.go

```diff
@@ -28,6 +28,9 @@ import (
 // from false to true).
 type ConditionFunc func(event Event) (bool, error)
 
+// max time.Duration value, around 290 years
+const Forever time.Duration = 1<<63 - 1
```

Comment:
I agree. This looks different right now: https://github.com/kubernetes/kubernetes/blob/master/pkg/watch/until.go#L43